rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,172 @@
+### DCP PR:
+
+# RFC Process
+
+## Summary
+
+The Human Cell Atlas Data Coordination Platform (HCA DCP) evolved informal, consensus-driven processes to coordinate and document technical and governance decisions.
+
+This early *lightweight* and *egalitarian* approach enabled the DCP to achieve rapid velocity and incorporate a diverse set of innovative design concepts.
+
+As DCP continues to mature and scale, our community can more easily contribute and learn when there is a formal and consistent **Request for Comments (RFC)** process to propose enhancements to software or governance which are then maintained in a central repository.
+
+## Author(s)
+*DCP PM*
+
+**Note:** *This proposal is based on ideas from the **HCA DCP Technical Decision-Making** white paper written by Bruce Martin with contributions from Tony Burdett, Laura Clarke, Brian O’Connor, Brian Raymor, and Tim Tickle.*
+
+## Shepherd
+[Brian Raymor](brianraymor@chanzuckerberg.com)
+
+## Motivation
+
+There are a number of limitations with our current process which need to be addressed:
+
+- For community members who want to propose enhancements to either technical software or governance, it is often unclear where to start - *for example, which HCA slack channel or project member to approach with a proposal.*
+
+  - There is no common template for design proposal submissions.
+
+  - There is no common process describing how the community reviews and approves submitted proposals.
+
+  - There is no formal assistance to guide a contributor through the process. 
+
+- New community members experience difficulties when _onboarding_. There is no curated repository of up-to-date design documents to assist with learning about the software. 
+
+- To ensure consistency of the architectural design, senior technical leaders must monitor multiple work streams or hope that they are notified of potential technical changes - *this is a significant concern for cross-cutting issues such as security or the data model*.
+
```

Comment:
I would add another reason, for box/project leadership there is no formal way of tracking major feature decision making process.  What I mean by that is, if I want to keep up with say blue box, I need to minimally attend the sprint planning and demo meetings, backlog grooming, architects meeting, and then keep up to date on multiple slack channels.  By moving an important decisions/features to RFCs I know, as a box lead, I can read the RFCs and see the most important discussions in a single place.  This hugely streamlines the "n x n" information problem in large projects like HCA (where everyone feels like they need to communicate with all other folks on the project via meetings/slacks/emails in order to stay "on top" of the project but ultimately don't get work done because they've spent all their time communicating).  With teams leaning into RFCs for any significant interface/feature designs, it will greatly streamline information flow in the project.  At least I think this will be true.